src/main/java/com/example/demo/model/CreditCard.java

```diff
@@ -0,0 +1,51 @@
+package com.example.demo.model;
+
+import com.example.demo.dto.CardDTO;
+import com.sun.istack.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.Where;
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Table(name = "credit_cards")
+@SQLDelete(sql = "UPDATE credit_cards SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted = false")
+public class CreditCard {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @NotNull private Long user_id;
```

Comment:
```suggestion
  @NotNull private Long userId;
```
[Java Naming Conventions](https://www.geeksforgeeks.org/java-naming-conventions/)

Reply:
fixed!